sycl/test-e2e/VirtualMem/memory_granularity.cpp

```diff
@@ -0,0 +1,76 @@
+// REQUIRES: aspect-ext_oneapi_virtual_mem
```

Comment:
It probably make sense to outline that requirement into `lit.local.cfg`. See `VirtualFunctions` and `ESIMD` folders for examples